src/Tizen.WebView/Tizen.WebView/PolicyDecision.cs

```diff
@@ -0,0 +1,183 @@
+        /// Download.
+        /// </summary>
+        Download,
+        /// <summary>
+        /// Ignore.
+        /// </summary>
+        Ignore
+    };
+
+    /// <summary>
+    /// This class provides the properties for Policy Decision of WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class PolicyDecision
+    {
+        private IntPtr _handle;
+
+        internal PolicyDecision(IntPtr handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Gets the Url.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Url
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_url_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets the scheme.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Scheme
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_scheme_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets response status code.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Cookie
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_cookie_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets the policy navigation type.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public PolicyNavigationType NavigationType
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_navigation_type_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets the policy decision type.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public PolicyDecisionType DecisionType
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_type_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets response status code.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int ResponseStatusCode
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_response_status_code_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Checks if frame requested in policy decision is main frame.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsMainFrame()
+        {
+            return Interop.ChromiumEwk.ewk_policy_decision_is_main_frame(_handle);
+        }
+
+        /// <summary>
+        /// Ignores the action which triggers this decision.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool Ignore()
```

Comment:
It should be void, in C# we don't return boolean on success.

Reply:
Done